<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор языка</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{border-image:url(background-image/back_4.jpg)}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="history_lang">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Информация о выбранном языке</string>
    </property>
   </widget>
   <widget class="QComboBox" name="list_of_languages">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox::drop-down{
    border:                 none;
    background-color:   rgb(87, 96, 134);
    color:                      rgb(255, 255, 255);
    font-weight:            bold;
    padding:                    0px;
}</string>
    </property>
   </widget>
   <widget class="QToolButton" name="selected">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/selected.png);
	icon-size: 140 px;
	border: None;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="company_name">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>520</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Scantype Black PERSONAL USE</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string> k h a i r u l l o v</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>© 2021–2021</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_google">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/google.png);
	icon-size: 10 px;
	border: None;
}
QToolButton:pressed{
  image: url(icons/selected.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_tg">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>520</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/tg.png);
	icon-size: 10 px;
	border: None;
}
QToolButton:pressed{
  image: url(icons/selected.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="contact_vk">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>29</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/vk.png);
	icon-size: 10 px;
	border: None;
}
QToolButton:pressed{
  image: url(icons/selected.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	image: url(icons/logo.png);
	icon-size: 10 px;
	border: None;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
